custom/conf/app.ini.sample

```diff
@@ -308,6 +308,9 @@ ACTIVE_CODE_LIVE_MINUTES = 180
 RESET_PASSWD_CODE_LIVE_MINUTES = 180
 ; Whether a new user needs to confirm their email when registering.
 REGISTER_EMAIL_CONFIRM = false
+; List of domain names that are allowed to be used to register on a Gitea instance
+; gitea.io example.com
+EMAIL_DOMAIN_WHITELIST=
```

Comment:
It should be coma or semicolon separated same as we have in other settings

Reply:
Fixed in https://github.com/go-gitea/gitea/pull/5157/commits/4fdd1aea44222a3b74de2f69bd5c96a1e783dd47